installer/conf/rsyslog-lad.conf

```diff
@@ -0,0 +1,2 @@
+# OMS Syslog collection for workspace LAD (SYSLOG_PORT=%SYSLOG_PORT%)
```

Comment:
To have these `%SYSLOG_PORT%` and other similar strings replaced, you will eventually need logic in `omsadmin.sh` or other script to replace them in these files.